app/src/main/java/com/example/medrecord/RecordManager.java

```diff
@@ -0,0 +1,63 @@
+package com.example.medrecord;
```

Comment:
This file has several errors too. Please help look into them. Thank you for the effort you are putting into this thing.

Reply:
Resolved. Check it out now